### PR TITLE
TASK-006 Adding Django Models and Admin

### PR DESCRIPTION
**Pull request by victormark97**
Added models for SoftDeletion, Image, DjProfile, Location, Event, Song and SongRequest

### CODE REVIEW

File: bethedj/settings.py [hunk 1]
```diff
@@ -32,8 +33,9 @@
 ]
 
 
-LOGIN_REDIRECT_URL = 'home'
+LOGIN_REDIRECT_URL = 'accounts/login'
```
Comment (victormark97) [inline bethedj/settings.py:36]: this is wrong

File: app/migrations/0001_initial.py [hunk 2]
```diff
@@ -21,7 +21,6 @@ class Migration(migrations.Migration):
                 ('is_superuser', models.BooleanField(default=False, help_text='Designates that this user has all permissions without explicitly assigning them.', verbose_name='superuser status')),
                 ('email', models.EmailField(max_length=254, unique=True)),
                 ('name', models.CharField(max_length=50)),
-                ('is_dj', models.BooleanField(default=False)),
```
Comment (victormark97) [inline app/migrations/0001_initial.py:24]: normally illegal to change a past migration, but this is still initial phase so accepted

**Review by paulmark03 — APPROVED** [review]
lgtm

**Review by paulmark03 — APPROVED** [review]
lgtm

**Review by paulmark03 — APPROVED** [review]
lgtm

**Review by paulmark03 — APPROVED** [review]

**Review by paulmark03 — APPROVED** [review]

**Pull request merged**